src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -907,7 +974,9 @@ void ImageReplayer<I>::handle_process_entry_safe(const ReplayEntry& replay_entry
     return;
   }
 
-  m_remote_journaler->committed(replay_entry);
+  if (m_remote_journaler) {
```

Comment:
Yes, it happened. That is why I had to check if it still exists. Do you think it's an erroneous behavior?